doc/symfony6.md

```diff
@@ -0,0 +1,445 @@
+[OK]   Gedmo\Translatable\Entity\MappedSuperclass\AbstractTranslation
+```
+
+This is very useful for advanced requirements and quite simple to understand. So now let's map
+everything the extensions provide:
+
+```yaml
+# only orm config branch of doctrine
+orm:
+    auto_generate_proxy_classes: %kernel.debug%
+    auto_mapping: true
+# only these lines are added additionally
+    mappings:
+        translatable:
+            type: annotation # or attribute
+            alias: Gedmo
+            prefix: Gedmo\Translatable\Entity
+            # make sure vendor library location is correct
+            dir: "%kernel.project_dir%/vendor/gedmo/doctrine-extensions/src/Translatable/Entity"
+        loggable:
+            type: annotation # or attribute
+            alias: Gedmo
+            prefix: Gedmo\Loggable\Entity
+            dir: "%kernel.project_dir%/vendor/gedmo/doctrine-extensions/src/Loggable/Entity"
+        tree:
+            type: annotation # or attribute
+            alias: Gedmo
+            prefix: Gedmo\Tree\Entity
+            dir: "%kernel.project_dir%/vendor/gedmo/doctrine-extensions/src/Tree/Entity"
+```
+<a name="ext-filtering"></a>
+## Filters
+
+The **softdeleteable** ORM filter also needs to be configured, so that soft deleted records are filtered when querying.
+To do so, add this filter info to your **doctrine.orm** configuration, edit **config/doctrine.yaml**:
+```yaml
+doctrine:
+    dbal:
+# your dbal config here
+    orm:
+        auto_generate_proxy_classes: %kernel.debug%
+        auto_mapping: true
+# only these lines are added additionally
+        filters:
+            softdeleteable:
+                class: Gedmo\SoftDeleteable\Filter\SoftDeleteableFilter
+```    
+<a name="ext-listeners"></a>
+
+## Doctrine extension listener services
+
+Next, the heart of extensions are behavioral listeners which pours all the sugar. We will
+create a **yaml** service file in our config directory. The setup can be different, your config could be located
+in the bundle, it depends on your preferences. Edit **config/packages/doctrine_extensions.yaml**
+
+```yaml
+# services to handle doctrine extensions
+# import it in config/packages/doctrine_extensions.yaml
+services:
+    # Doctrine Extension listeners to handle behaviors
+    
+    # Symfony 4 example
+    gedmo.listener.timestampable:
+        class: Gedmo\Timestampable\TimestampableListener
+        tags:
+            - { name: doctrine.event_subscriber, connection: default }
+        calls:
+            - [ setAnnotationReader, [ "@annotation_reader" ] ]
+
+    # Symfony 6.3 change          
+    gedmo.listener.timestampable:
+        class: Gedmo\Timestampable\TimestampableListener
+        tags:
+            - { name: doctrine.event_listener, event: 'prePersist', connection: default }
+            - { name: doctrine.event_listener, event: 'onFlush', connection: default }
+            - { name: doctrine.event_listener, event: 'loadClassMetadata', connection: default }
```

Comment:
This isn't optimal, the correct way to go about this would be to stop inheriting from MappedEventSubscriber in https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/src/AbstractTrackingListener.php and changing the code entirely, but that's unfortunately a little past my familiarity with the project.